netsync/manager.go

```diff
@@ -389,14 +389,47 @@ func (sm *SyncManager) handleNewPeerMsg(peer *peerpkg.Peer) {
 	}
 }
 
+// handleCheckSyncPeer selects a new sync peer.
+func (sm *SyncManager) handleCheckSyncPeer() {
+	if atomic.LoadInt32(&sm.shutdown) != 0 {
+		return
+	}
+
+	log.Debugf("Checking Sync Peer, time since last block: %v", sm.lastBlockTime)
```

Comment:
I don't think I would log this. Might load up the logs with this line and make it harder to read. Maybe only log if the threshold is hit.

Reply:
Will fix, I figured as debug it would be ok. =)